config/config.exs

```diff
@@ -53,3 +53,8 @@ config :phoenix, :json_library, Jason
 # Import environment specific config. This must remain at the bottom
 # of this file so it overrides the configuration defined above.
 import_config "#{config_env()}.exs"
+
+# Google auth variables
+config :elixir_auth_google,
+  google_scope:
+    "profile email https://www.googleapis.com/auth/calendar https://www.googleapis.com/auth/calendar.events https://www.googleapis.com/auth/calendar.events.readonly https://www.googleapis.com/auth/calendar.readonly https://www.googleapis.com/auth/calendar.settings.readonly"
```

Comment:
are these scopes additive? 
does `https://www.googleapis.com/auth/calendar` not supersede all the rest? 💭

Reply:
I don't believe so. I've followed the recommendations from https://developers.google.com/identity/protocols/oauth2/scopes#calendar.

<img width="905" alt="image" src="https://user-images.githubusercontent.com/17494745/235891770-ffef3b58-b115-4055-a5a9-6e9cf4190d93.png">

Reply:
What I mean is: in terms of permissions models the gradual refinement _reduces_ the level of permissions.
i.e. `https://www.googleapis.com/auth/calendar` gives the _complete_ access to the person's calendar
and therefore the `X.readonly` is redundant. 
If we wanted to follow the principal of _least_ privilege then we would only request the permissions we _absolutely_ require: `https://www.googleapis.com/auth/calendar.events` i.e. "**`View` and `edit` events on all your calendars**"

and not have the ability to "***permanently `delete` all the calendars***" 🔥 

<img width="907" alt="google-calendar-permissions" src="https://user-images.githubusercontent.com/194400/235894367-5d351c31-299d-4852-ab10-cebd53455942.png">

In the wrong hands either of these permissions levels could be destructive.
But the "**_Full_ Access**" `https://www.googleapis.com/auth/calendar` is almost _never_ appropriate. 💭

Reply:
You're correct. I assumed that since `events` only live in `calendars` (these are resources), the user would need to consent access to calendars and *then* to events. I didn't think that having access to calendar events would assume access to calendar (as counter-intuitively as that sounds, since `calendar.events` is a scope in of itself).

In that case, only `calendar.events` makes sense in this project.

Reply:
Cool. I will update it now on the branch. 👍

Reply:
But totally agree the docs are unnecessarily confusing. 🙄